content/docs/tech-insights/define-custom-data-sources/index.md

```diff
@@ -122,6 +122,14 @@ If you want to parse YAML content from plain strings, for example from an API En
 * JSONata query: `$parseYaml(spec.definition).openapi`
 * Type: String
 
+**$getEntityData**
+
+If you want to parse request entity information in the JSONata context you can use the `$getEntityData` function. An example query could be like the following:
+* Data Provider Type: Entity Definition
+* Fact Name: OpenAPI Spec Version
```

Comment:
```suggestion
* Fact Name: First Target Ref of relations
```

Reply:
What does fact name mean?

Reply:
it depends what kind of example you want to display within this block. Judging from the query below, it is not getting OpenAPI spec version

Reply:
Ah I see what you mean.

Reply:
How about that.